jetty-client/src/main/java/org/eclipse/jetty/client/http/HttpChannelOverHTTP.java

```diff
@@ -39,6 +40,8 @@
     private final HttpConnectionOverHTTP connection;
     private final HttpSenderOverHTTP sender;
     private final HttpReceiverOverHTTP receiver;
+    private final LongAdder _inMessages = new LongAdder();
```

Comment:
inconsistent field names.  As much as I like `_fieldName`, this class has been written with just `fieldName`, so you should match the class.